brainstormforce/astra
# AST-2960 - WordPress 6.2 Beta Compatibility - Showing gap in between eye icon and title in editor

### Description

- Showing gap in between eye icon and title in editor.
- Activating on hover.

### Screenshots

https://d.pr/i/fUyAqZ

### Types of changes


fix: Showing gap in between eye icon and title in editor.
fix: - Activating on hover.



### How has this been tested?

- Create new post or edit existing.
- Click on post title (it should activate on click and not on hover).
- Visibility icon should be properly aligned.

### Checklist:
